docs/DeveloperGuide.adoc

```diff
@@ -780,19 +780,27 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 [appendix]
 == Product Scope
 
-*Target user profile*: Couples in NUS who:
-
-* want to plan dates, in and out of school
-* want to store their memories via private journal entries
+*Target user profile:*
+
+* is currently a student of NUS
+* is in a relationship with a student of NUS
+* may have to travel to unfamiliar locations in NUS
+* want to remember special moments
+* want to remember special dates
+* want to schedule meetings
 * prefer desktop apps over other types
 * can type fast
 * prefer typing over mouse input
 * are reasonably comfortable using CLI apps
 
-*Value proposition*: Help couples in NUS to create new memories and remember them through:
+*Value proposition:* all-in-one desktop app to help NUS couples make and remember memories with each other
+
+*Feature contribution:*
```

Comment:
Maybe can just write "Features:"

Reply:
They explicitly asked for a subsection labeled "Feature contribution" though. Not sure whether we can arbitrarily change the name.